addOns/ascanrules/src/test/java/org/zaproxy/zap/extension/ascanrules/GetForPostScanRuleUnitTest.java

```diff
@@ -83,4 +144,21 @@ void shouldHaveExpectedExampleAlerts() {
     public void shouldHaveValidReferences() {
         super.shouldHaveValidReferences();
     }
+
+    private String getResponseForPostRequest(String time) {
```

Comment:
`static`, same for following.

Reply:
im curious, why do we use static here? is it a convention followed or does it have any other benefits while running tests?

Reply:
Just convention.